src/commands/demo-config/create.ts

```diff
@@ -0,0 +1,126 @@
+import fs from "fs";
+import { error, info, success } from "signale";
+import { Argv } from "yargs";
+import { deployDocumentStore } from "../../implementations/deploy/document-store/document-store";
+import { deployTokenRegistry } from "../../implementations/deploy/token-registry/token-registry";
+import { readFile } from "../../implementations/utils/disk";
+import { create as CreatedWalletPath } from "../../implementations/wallet/create";
+import { getLogger } from "../../logger";
+import { highlight } from "../../utils";
+import { handler as CreateTemporaryDns } from "../dns/txt-record/create";
+import ConfigTemplate from "./config-template.json";
+import { CreateConfigCommand } from "./config.type";
+
+interface ConfigFile {
+  wallet: string;
+  forms: {
+    defaults: {
+      issuers: {
+        name: string;
+        documentStore?: string;
+        tokenRegistry?: string;
+        identityProof: {
+          type: string;
+          location?: string;
+        };
+      }[];
+    };
+  }[];
+}
+
+const { trace } = getLogger("demo-config:create");
+
+export const command = "create [options]";
+
+export const describe = "Create a demo config file";
+
+export const builder = (yargs: Argv): Argv =>
+  yargs
+    .option("output-dir", {
+      alias: "od",
+      description: "Write output to a directory",
+      type: "string",
+      demandOption: true,
+    })
+    .option("encrypted-wallet-path", {
+      type: "string",
+      description: "Path to file containing wallet.json",
+      normalize: true,
+      demandOption: false,
+    });
+
+export const handler = async (args: CreateConfigCommand): Promise<void> => {
+  trace(`Args: ${JSON.stringify(args, null, 2)}`);
+  if (!fs.existsSync(args.outputDir)) {
+    fs.mkdirSync(args.outputDir);
+  }
+
+  try {
+    let walletPath = "";
+    if (!args.encryptedWalletPath) {
+      info(`Please enter password to create wallet`);
+      const createWalletParams = {
+        outputFile: `${args.outputDir}/wallet.json`,
+        fund: "ropsten",
+      };
+      walletPath = await CreatedWalletPath(createWalletParams);
+      info(`Wallet created at ${walletPath}`);
+    }
+
+    const walletFile = walletPath ? walletPath : args.encryptedWalletPath;
+    const wallet = await readFile(walletFile);
+
+    info(`Enter password to continue deployment of Document Store`);
+    const deployDocumentStoreParams = {
```

Comment:
let's skip this if the config file does not need one

Reply:
ok updated so that the user will be prompted to create with a verifiable document or transferable records.